src/App.js

```diff
@@ -1,23 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import Header from "./components/Header";
 
 function App() {
   return (
     <div className="App">
```

Comment:
can you please remove the app class from the div and move the styling to the specific component.